### DISCUSSION TITLE
Multiple users

### DISCUSSION THREAD

**Discussion by Lockhart3195GC** [moraroy/NonSteamLaunchers-On-Steam-Deck#107] [Q&A] [answered]
So I downloaded and have had no issues launching any launcher I downloaded but when I switch accounts to my wife's none of the launchers show up in the library. Specifically I'm trying to get xbox game pass to show up.

**Comment by moraroy** [comment]
@Lockhart3195GC interesting OK I will look into this thank you.

**Reply by csmende** [reply]
@moraroy New Steam Deck user here - love this! I am having the same issue on my deck. The non-steam library entries only show up for the first user added to the deck. Both users exist when the install was attempted. I have reproduced this by resetting my deck and adding reversing the order of the two accounts added - in both cases only the first account can see the library entries. Epic, Minecraft, GOG are the ones I used.

**Comment by moraroy** [comment, marked as answer]
Fixed
